docs/how to/use-transferbench.rst

```diff
@@ -36,71 +39,102 @@ This simplifies to a simple copy operation when dealing with single SRC/DST.
 
 Three Executors are supported by TransferBench::
```

Comment:
Is Executors supposed to have initial cap?

Reply:
Corrected.